src/TracksLayer/index.js

```diff
@@ -35,12 +35,13 @@ const TracksLayer = ({ onPointClick, showTimepoints }) => {
 
   return subjectTracksWithPatrolTrackShownFlag.length > 0
     ? subjectTracksWithPatrolTrackShownFlag.map((subjectTracks) => <TrackLayer
-        key={`track-layer-${subjectTracks.track.features[0].properties.id}`}
-        linePaint={{ 'line-opacity': subjectTracks.patrolTrackShown ? 0.4 : 1 }}
-        onPointClick={onTimepointClick}
-        showTimepoints={showTimepoints}
-        trackData={subjectTracks}
-      />)
+      id={subjectTracks.track.features[0].properties.id}
```

Comment:
This is the culprit of the bug I suppose. 
Looks good to me overall!

Not knowing the exact business logic, a question comes to mind when reviewing this:
Is it possible to have an empty `features` list? In that case, how do we handle it?

Reply:
Good question! By the time the selector chain gets to this point, the data has been pretty thoroughly cleaned and prepared. I don't think it's possible to hit this point having a `track` property but without a linestring feature inside that track.